transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/ListTransferMethodFragment.java

```diff
@@ -162,13 +162,6 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         super.onViewCreated(view, savedInstanceState);
         mProgressBar = view.findViewById(R.id.list_transfer_method_progress_bar_layout);
         mEmptyListView = view.findViewById(R.id.empty_transfer_method_list_layout);
-        Button button = view.findViewById(R.id.add_account_button);
-        button.setOnClickListener(new OneClickListener() {
-            @Override
-            public void onOneClick(View v) {
-                mOnAddNewTransferMethodSelected.showSelectTransferMethodView();
-            }
-        });
```

Comment:
Are the UI tests passing after removing this?

Reply:
UI tests are passing after this. Please find the screenshot. 
<img width="1249" alt="Screenshot 2021-02-26 at 4 05 06 PM" src="https://user-images.githubusercontent.com/72095964/109289606-745c8480-784c-11eb-8f04-68d4fcde0d8b.png">